src/aims_ui/api_interaction.py

```diff
@@ -1,33 +1,40 @@
+import os
+import json
 import requests
 from . import app
+from flask import render_template
+from .models.get_endpoints import get_endpoints
 
+def get_params(all_user_input):
+  """Return a list of aparameters formatted for API header, from class list of inputs"""
+  params = 'verbose=True&&'
+  for param in all_user_input.keys():
+    # Do not add epoch for testing
+    if (param != 'epoch') and (os.getenv('FLASK_ENV') == 'development'):
+      if str(all_user_input.get(param)) != '':
+        params = params + str(param) +'='+ str(all_user_input.get(param)).replace('%','') + '&&'
+
+  params = params[:-2] if params != '' else ''
+
+  return params
 
 def api(
     url,
     called_from,
-    all_user_input,
-):
+    all_user_input):
+  """API helper, all pages go through here to interact with API"""
+
+  header = {"Content-Type": "application/json",}
 
   if (called_from == 'uprn') or (called_from == 'postcode'):
     url = app.config.get('API_URL') + url + all_user_input.get(called_from)
-    r = requests.get(url, )
+    params = get_params(all_user_input)
+    r = requests.get(url, params=params, headers=header, timeout=1000000., verify=False)
```

Comment:
I wouldn't suggest setting a timeout this high, can this be dropped down to a more balanced number?

Reply:
It's also generally not a good idea to disable verification, and inter-cluster messages don't even use HTTPS anyway.

Reply:
Ill remove verify and remove timeout leaving it as the default